activities.el

```diff
@@ -801,6 +867,63 @@ activity's name is NAME."
                   "In the meantime, it's recommended to not use buffers of this major mode in an activity's layout; or you may simply ignore this error and use the other buffers in the activity.")
           (current-buffer)))))
 
+(defvar activities--age-spec
+  `((?Y "year"   "years"   ,(round (* 60 60 24 365.2425)))
+    (?M "month"  "months"  ,(round (* 60 60 24 30.436875)))
+    (?w "week"   "weeks"   ,(* 60 60 24 7))
+    (?d "day"    "days"    ,(* 60 60 24))
+    (?h "hour"   "hours"   ,(* 60 60))
+    (?m "min"    "mins"    60)
+    (?s "sec"    "secs"    1))
+  "Age specification.  See `magit--age-spec', which this duplicates.")
+
+(defun activities--age (age &optional abbrev)
+  "Summarize AGE.
+Abbreviate the units if ABBREV is non-nil.  Based on
+`magit--age'."
+  ;; TODO: replace this if seconds-to-string adds READABLE support
```

Comment:
Does this TODO need to be updated?  Also, a link to the relevant Emacs bug should be included.

Reply:
Not sure if you are following that bug (#[71572](https://yhetil.org/emacs-bugs/D91AFAEE-C909-4238-AFF2-226727D4B0F6@gmail.com/T/#t)), but some expressed concern about including a unicode char.  I don't personally think that's an issue in this day and age, but wanted to check.

Reply:
I've been following it but not very closely.  I didn't realize how much of a bikeshed it was going to become (not a hostile one--I'm just surprised at how long it's gone on).  :)  Maybe that's why there are several implementations floating around, because everyone has their own preferences.

I don't have a problem with using Unicode here, as long as the character is readable, and not only present in very recent versions of common fonts.

Reply:
I fully expected the bike-shedding ;).  We'll see how it converges; it appears it's a problem that's been solved many times.  As long as something reasonably flexible appears in Emacs 31 it probably doesn't matter much.  If it goes in without the ½, which seems likely, we can always give precision = 0.5 and modify `x.5` to read `x½`.